group.py

```diff
@@ -2,4 +2,7 @@
 
 # Your code to go here...
 
-my_group =
+my_group = [{'Name': 'Jill', 'Age':26, 'job':'biologist','connections': {'friend':'Zalika', 'partner':'John'}}, 
+           {'Name': 'Zalika', 'Age':28, 'job':'artist','connections':{'landlord':'Nash','friend':'Jill'}},
+           {'Name': 'John', 'Age':27, 'job':'writer','connections':{'cousin':'Nash','partner':'Jill'}},
+           {'Name': 'Nash', 'Age':34, 'job':'chef','connections':{'cousin':'John','tenant':'Zalika'}}]
```

Comment:
Good, could possibly link connections between people to other places within the dictionary